tgui/packages/tgui/interfaces/GeneTek/tabs/ScannerTab.js

```diff
@@ -15,6 +15,7 @@ import { GeneIcon } from "../GeneIcon";
 export const ScannerTab = (props, context) => {
   const { data, act } = useBackend(context);
   let [changingMutantRace, setChangingMutantRace] = useSharedState(context, "changingmutantrace", false);
+  const [preview, previewToggle] = useSharedState(context, 'previewToggle', false);
```

Comment:
Would have preferred `togglePreview` here but not going to hold up the PR on it.

Reply:
*sneaks that into resolving a merge conflict*